unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackEditorHUDController.cs

```diff
@@ -339,38 +350,41 @@ private void LoadUserProfile(UserProfile userProfile)
                 return;
             }
 
-            async UniTaskVoid LoadUserProfileAsync(UserProfile userProfile, CancellationToken cancellationToken)
+            async UniTaskVoid LoadUserAvatarAsync(AvatarModel avatar, CancellationToken cancellationToken)
             {
                 try
                 {
-                    foreach (string w in userProfile.avatar.wearables)
```

Comment:
Why this got deleted? How are we storing the extended urns for the wearables?

Reply:
This was a double loop that could be improved. I moved this piece inside another loop down below on line 384 and 385